ui/src/components/media/calling.rs

```diff
@@ -490,6 +494,11 @@ fn PendingCallDialog(cx: Scope<PendingCallProps>) -> Element {
             }
         };
     }
+    let alive = use_ref(cx, || Arc::new(AtomicBool::new(false)));
+    use_effect(cx, (), |_| {
```

Comment:
the `use_effect` needs to re-run for subsequent calls. the call id should be passed as a dependency to ensure this happens.

Reply:
i dont see a reason as to why though? you can only have one active call or an incoming call.

Reply:
I was thinking the same `use_effect` would be used across multiple instantiations of the `Element`. but that's probably not how it works. comment withdrawn.

Reply:
`use_effect` gets run for every new instance. but not for rerenders